superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml

```diff
@@ -69,6 +69,10 @@ superset_docker_restart_policy: "Always"
 superset_service_port: "32443"
 superset_nginx_port: "32443"
 
+SUPERSET_NPROCESSORS: 1
```

Comment:
As we earlier discussed, all new variables would be prefixed with RVF instead of SUPERSET but lets not pick this in the current PR, but we would convert every variable name prefixed to RVF in separate PR.